server/plugins/schema-plugin-ob-mysql/src/test/java/com/oceanbase/odc/plugin/schema/obmysql/parser/OBMySQLGetDBTableByParserTest.java

```diff
@@ -230,7 +230,7 @@ public void getPartition_secondary_key_no_template_Success() {
         Assert.assertEquals("col3", partition.getSubpartition().getPartitionOption().getColumnNames().get(1));
         Assert.assertEquals(DBTablePartitionType.KEY, partition.getSubpartition().getPartitionOption().getType());
         Assert.assertEquals(2, partition.getSubpartition().getPartitionOption().getColumnNames().size());
-        Assert.assertTrue(partition.getSubpartition().getPartitionOption().getPartitionsNum() == 3);
+        Assert.assertNull(partition.getSubpartition().getPartitionOption().getPartitionsNum());
```

Comment:
What's this change for?

Reply:
Non-template subpartition should not have partitionNum
![image](https://github.com/user-attachments/assets/8686a1ea-ef98-43e8-8bb6-86c3c1f3805d)